### DISCUSSION TITLE
How to deploy a macos application

### DISCUSSION THREAD

**Discussion by youfun** [liveview-native/liveview-client-swiftui#1442] [General]
i use xcode makd a installer package, but run with error.  do i need run" MIX_ENV=prod mix release" first,
<img width="587" alt="image" src="https://github.com/user-attachments/assets/84659ce2-7b6f-499d-b29b-f5922388602f">
i click copy error ,but found nothing

**Comment by bcardarella** [comment]
If Copy Error is empty that is a bug /cc @carson-katri 

As far as the issue at-hand, if you are building an installation package you probably need to set the `production` url instead of `development` in your app's `ContentView.swift` https://github.com/liveview-native/liveview-client-swiftui/blob/main/priv/templates/lvn.swiftui.gen/xcodegen/TemplateApp/ContentView.swift#L15

**Reply by youfun** [reply]
thanks reply. The demo application utilizes a local SQLite database and does not require an internet connection to function. In this scenario, how should I proceed?

**Reply by bcardarella** [reply]
@youfun are you saying the Phoenix app is using sqlite or the compiled LVN app is using sqlite?

**Reply by youfun** [reply]
both of them using sqlite  


```elixir

# in config/dev.exs 
config :my_app, MyApp.Repo,
  database: Path.expand("../my_app,_dev.db", Path.dirname(__ENV__.file)),
  pool_size: 5,
  stacktrace: true,
  show_sensitive_data_on_connection_error: true

#in config/runtime.exs

if config_env() == :prod do
  database_path =
    System.get_env("DATABASE_PATH") ||
    Path.join(File.cwd!(), "my_app.db")


```

**Reply by bcardarella** [reply]
@youfun I'm sorry but I need far more context here. For the time being LiveView Native requires a connection to the Phoenix server whether that is on the Internet in production or locally for development.

I'm not quite sure what it is you're trying to do and you haven't provided enough context or information for me to answer any of your questions.

**Reply by youfun** [reply]
This project solely utilizes SQLite as its local database and operates entirely on the local machine without the need to connect to any online servers. so i don't konw how to set the production url , just like run a phoenix app on local pc. the url maybe "http://127.0.0.1:4000"
